images/kubeadm/runner

```diff
@@ -21,14 +21,6 @@ if [ ! -e test-infra ]; then
   git clone https://github.com/kubernetes/test-infra
 fi
 
-if [ ! -e kubernetes-anywhere ]; then
-  git clone https://github.com/kubernetes/kubernetes-anywhere
-
-  # Explicitly version this dependency so that upstream commits can't
-  # immediately break e2e jobs, and we have control over upgrading/downgrading.
-  git -C kubernetes-anywhere checkout a26f42c716028c4ba967b2d2f3de5ed50acc51da
-fi
-
 # This is required until https://github.com/kubernetes/kubernetes-anywhere/issues/332
 # is implemented.
 if [ ! -e kubernetes-anywhere/phase1/gce/account.json ]; then
```

Comment:
@BenTheElder you've mentioned that `--repo` would pull in `$PWD/../kubernetes-anywhere`
do i need to adjust the paths in this last part of the file to be `../kubernetes-anywhere/`?

Reply:
i also saw the discussion about not having a custom image.

i'm not exactly sure why the `ln -s` action is here and not in the kubernetes-anywhere tree, but is it possible to move the `./test-infra/jenkins/bootstrap.py` call in the job?

this can go in a separate PR too, i guess.

Reply:
So when the init scripts hand off to the testing code the current directory will be a checkout of the first --repo flag, which is typically the repo under test. Additional repos will be adjacent in ./../$OTHER_REPO.

I'm not sure why we do anything in this image. Ideally we'd use the underlying runner in the base image only and not do anything at runtime in this image.